pudb/var_view.py

```diff
@@ -347,6 +348,89 @@ class ValueWalker:
     def __init__(self, frame_var_info):
         self.frame_var_info = frame_var_info
 
+    def add_continuation_item(self, parent, id_path, count):
+        """
+        :param VariableWidget or None parent
+        :param str id_path:
+        :param int count:
+        :returns: True if a continuation item ("...") was added, else False
```

Comment:
Explain the logic better here. I'm guessing "a continuation item was not added" means that the calling walk should continue?